### PR TITLE
logging: start_time and end_time are not required params

### PR DESCRIPTION
**Pull request by huanggze**
Signed-off-by: huanggze <loganhuang@yunify.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/simple/client/logging/elasticsearch/api_body.go [hunk 1]
```diff
@@ -119,14 +119,15 @@ func (bb *bodyBuilder) mainBool(sf logging.SearchFilter) *bodyBuilder {
...
 
-	if !sf.Starttime.IsZero() || !sf.Endtime.IsZero() {
-		fromTo := Match{
-			Range: &Range{&Time{
-				Gte: &sf.Starttime,
-				Lte: &sf.Endtime,
-			}},
-		}
-		ms = append(ms, fromTo)
+	r := &Range{Time: &Time{}}
```
Comment (zryfish) [inline pkg/simple/client/logging/elasticsearch/api_body.go:122]: please don't use single letter named variable, it's not a good practice. Use meaningful names.

Comment (huanggze) [reply pkg/simple/client/logging/elasticsearch/api_body.go:122]: It's a local variable. Besides, range is keyword

image

image

**Pull request merged**